sdk/file_sdk/Common/Common.pro

```diff
@@ -4,7 +4,13 @@ TARGET    = modfilecommon
 
 TEMPLATE  = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
+
 INCLUDEPATH += $$REPO_ROOT/sdk/file_sdk
+win32:INCLUDEPATH += $$REPO_ROOT/third_party/include/Libgsf
+
+win32:LIBS += -L$$REPO_ROOT/third_party/lib/Libgsf -llibgsf-1-114 -lzlib1\
+               -llibgobject-2.0-0\
```

Comment:
you broke linux build: fatal error: gsf/gsf-fwd.h: No such file or directory

Reply:
and scons I think, fixed for scons lets see if will fix linux one.

---
In reply to: [139358188](https://github.com/AzureAD/rms-sdk-for-cpp/pull/158#discussion_r139358188) [](ancestors = 139358188)